### ISSUE TITLE
Released version on PyPI is broken

### ISSUE THREAD

**Issue by ramonski** [senaite/senaite.core#988]
## Steps to reproduce

Add `senaite.core` to the eggs section in a new buildout environment

## Current behavior

Traceback occurs during install

## Expected behavior

Package get installed

## Traceback

```
While:
  Installing instance.
  Getting distribution for 'senaite.core>=1.2.7'.

An internal error occurred due to a bug in either zc.buildout or in a
recipe being used:
Traceback (most recent call last):
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/buildout.py", line 2128, in main
    getattr(buildout, command)(args)
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/buildout.py", line 798, in install
    installed_files = self[part]._call(recipe.install)
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/buildout.py", line 1558, in _call
    return f()
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/plone.recipe.zope2instance-4.3-py2.7.egg/plone/recipe/zope2instance/__init__.py", line 113, in install
    installed.extend(self.install_scripts())
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/plone.recipe.zope2instance-4.3-py2.7.egg/plone/recipe/zope2instance/__init__.py", line 620, in install_scripts
    requirements, ws = self.egg.working_set(['plone.recipe.zope2instance'])
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.recipe.egg-1.3.2-py2.7.egg/zc/recipe/egg/egg.py", line 101, in working_set
    **kw)
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/easy_install.py", line 957, in install
    return installer.install(specs, working_set)
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/easy_install.py", line 730, in install
    for dist in self._get_dist(req, ws):
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/easy_install.py", line 574, in _get_dist
    dists = [_move_to_eggs_dir_and_compile(dist, self._dest)]
  File "/Users/rbartl/develop/ridingbytes/senaite/senaite/eggs/zc.buildout-2.12.1-py2.7.egg/zc/buildout/easy_install.py", line 1771, in _move_to_eggs_dir_and_compile
    assert newdist is not None  # newloc above is missing our dist?!
AssertionError
```

**Comment by ramonski** [comment]
We need to do a `-post` release like in `senaite.api`: https://github.com/senaite/senaite.api/blob/master/setup.py#L5
But rather name the version than something like `1.2.8.post1`

**Comment by ramonski** [comment]
Released 1.2.8.2 on PyPI: https://pypi.org/project/senaite.core/1.2.8.2/
